stages/cloud-provider/aws/sharedsvc/iam.tf

```diff
@@ -54,6 +54,63 @@ module "cert_manager_iam" {
   openid_connect_provider_url = module.eks.aws_iam_openid_connect_provider.url
 }
 
+resource "aws_iam_role" "rode_service_account" {
```

Comment:
`cloud-provider` has already been applied to sharedsvc, but I wonder if this should be removed? I'm not seeing where it would be used.